### ISSUE TITLE
payment processor says "account insufficient funds".

### ISSUE THREAD

**Issue by frndxyz** [bonesoul/CoiniumServ#547]
i have 15 digitalcoin on wallet to transfer to miner but server shows info that account has not sufficient fund. is it because i use my miner address that is from same wallet as server wallet??

**Comment by djspacedevil** [comment]
Hy,

i have the same situation but the field is decimal(19,5)... So how can i fix it?

**Comment by gecox22** [comment]
I am having the same issue. My tables are set up correctly. I have rebuilt the pool numerous times. I have tried a new wallet checked the conf file for the wallet. I am mining DEM locally just to learn how pools work. When a block confirms I can see it is trying to make a payment for 49.54 and there is way more than that in the wallet. I only have one account in the wallet listed as Pool. The address is setup in the pool.json file the rewards portion of the .json file .1 goes to the pool wallet for free and .9 goes to a personal wallet. I have tried making everything come to the pool wallet and same issue happens. 

My tables are set as decimal with a 0 in the box so it looks like 19,8 and then 0.00000000

I have also completely wiped the Redis and the MYSQL Databases and started from scratch. I have also tried a completely new database as well. I am completely stump any help would be greatly appreciated.
